ChangeLog

```diff
@@ -4,6 +4,11 @@ ChangeLog
 Next version
 ------------
 
+*Bugfix:*
+
+    * Allows to override a configuration containing a dash.
```

Comment:
`configuration section or key containing [...]` ?

Et dans le message de commit : `convert dash *to* underscore`.